### PR TITLE
fix: Remove the text-emphasis from OG image 

### LINKED ISSUE #11219: Remove or replace text-emphasis from OG images
Found a bug? Please fill out the sections below. 👍

### Issue Summary

It seems like `text-emphasis` is not available in tailwind at the edge (where the OG images are being generated)

image

We should either include or remove those classnames.

https://github.com/calcom/cal.com/blob/main/packages/lib/OgImages.tsx#L142

### PR DESCRIPTION
**Pull request by Madhan13K**
Remove the text-emphasis from OG image

Fixes #11219 

- [ X] Bug fix (non-breaking change which fixes an issue)

### CODE REVIEW

File: apps/web/public/static/locales/en/common.json [hunk 1]
```diff
@@ -2037,5 +2037,6 @@
   "team_no_event_types": "This team has no event types",
   "seat_options_doesnt_multiple_durations": "Seat option doesn't support multiple durations",
   "include_calendar_event": "Include calendar event",
-  "ADD_NEW_STRINGS_ABOVE_THIS_LINE_TO_PREVENT_MERGE_CONFLICTS": "↑↑↑↑↑↑↑↑↑↑↑↑↑ Add your new strings above here ↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑"
+  "ADD_NEW_STRINGS_ABOVE_THIS_LINE_TO_PREVENT_MERGE_CONFLICTS": "↑↑↑↑↑↑↑↑↑↑↑↑↑ Add your new strings above here ↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑",
+  "email_address": "Email Address"
```
Comment (Udit-takkar) [inline apps/web/public/static/locales/en/common.json:2041]: why was this added ?

File: packages/features/bookings/lib/getBookingFields.ts [hunk 2]
```diff
@@ -251,6 +251,7 @@ export const ensureBookingInputsHaveSystemFields = ({
       type: "multiemail",
       editable: "system-but-optional",
       name: "guests",
+      defaultPlaceholder: "enter_address",
```
Comment (Udit-takkar) [inline packages/features/bookings/lib/getBookingFields.ts:254]: why did you add this ?

Comment (Madhan13K) [reply packages/features/bookings/lib/getBookingFields.ts]: I removed it can you please check now..
Thank You!

File: apps/web/public/static/locales/en/common.json (hunk 1 above)
Comment (Madhan13K) [reply apps/web/public/static/locales/en/common.json]: I have removed it, can you please check now...
